@here/olp-sdk-authentication/test/OAuth.test.ts

```diff
@@ -496,3 +499,44 @@ describe("auth-request-project-scope", () => {
     });
 
 });
+
+describe("expired token refreshing", async () => {
+    const mockedTokenRequester = async (params: any): Promise<Token> => {
+        const mockedToken: Token = {
+            accessToken: `${params.url}-${params.consumerKey}-${params.secretKey}`,
```

Comment:
Please, replace word "secret" to something else (veracode)

Reply:
It is our property from UserAuth.ts
```
.tokenRequester({
                url: this.m_apiUrl + "oauth2/token",
                consumerKey: this.m_credentials.accessKeyId,
                secretKey: this.m_credentials.accessKeyScrt,
                scope: this.m_scope
            })
```
Am I supposed to change it in both places?

Reply:
can we just remove "-${params.secretKey}" ?

Reply:
Sure. Done